requirements.dso.txt

```diff
@@ -0,0 +1,9 @@
+dacite
```

Comment:
I think we need dacite also from gardener-cicd-libs (ditto for component-model + github3 + requests + urllib3). as long as we keep all of our packages in the same repository, I think I would prefer to no duplicate those dependencies (-> double-maintenance, esp. if using version-pinning)